### PR TITLE
Replace StickyBM chromatic scale guide

### PR DESCRIPTION
**Pull request by Gankutsuou-Gank**
removed sticky cuz they are a scummy person lol, also added a Melodyne tutorial.

### CODE REVIEW

**Comment by ninjamuffin99** [comment]
can you provide more elaboration?

**Comment by ninjamuffin99** [comment]
also the link title should match the youtube video upload title! wahoo!

**Comment by doggogit** [comment]
I believe there was drama between Sticky and his friends, but as far as I know, they're in the clear.

**Comment by ninjamuffin99** [comment]
hmmm nothing is really giving me proper elaboration! 

victim of what?

**Comment by Gankutsuou-Gank** [comment]
> > > I believe there was drama between Sticky and his friends, but as far as I know, they're in the clear.
> > 
> > 
> > I dont think so, considering that the victim released a doc about sticky last week
> 
> Another one?

alex

**Comment by ninjamuffin99** [comment]
hmmm I feel like if i read a doc and youtube response video it's like this weird strange situation of me picking sides in some e-drama that I really don't care too much about if I'm being honest!

I think if there's something that's linked, that pretty BLATANTLY breaks something that's in the [Code of Conduct standards](https://github.com/FunkinCrew/funkin-resources/blob/main/CODE_OF_CONDUCT.md#our-standards) it should be removed.

As well as all of that, I do not want to be the single sole person who should be the judge of the situation! I'll use my judgement when necessary, but of course I will try to keep things less opinionated from me personally regarding this list! I think it should be COMMUNITY first and foremost, and if a majority of people would believe that this person has done something so terrible that they should be wiped from the internet, then so be it!

**Comment by ninjamuffin99** [comment]
I'm not saying that it breaks the Code of Conduct, mostly because... I still don't know what it is!

**Comment by Gankutsuou-Gank** [comment]
> I'm not saying that it breaks the Code of Conduct, mostly because... I still don't know what it is!

tldr lol
https://twitter.com/teh_hspeye/status/1617584566520410113?s=20

**Comment by Gankutsuou-Gank** [comment]
there is also this https://twitter.com/StickyBMleaks/status/1617253837332414466?s=20&t=mROZOSLCC6XIjjTFNJHL5Q

**Comment by Gankutsuou-Gank** [comment]
sticky is just a problematic person in general lol

**Comment by MrMercenary** [comment]
Hi ninjamuffin, this is Alexander Mercenary, the person directly involved. I am a victim of StickyBM, and have gone through extensive abusive and mistreatment by the hands of him and his team.

I don’t want to force you to feel as though you have to side with things, the last thing I want is to drag someone into drama they aren’t a part of. I just felt that it was best that I reach out in this way.

If you need to talk about anything regarding this, my Discord tag is Alexander Mercenary@3905

**Comment by ColdlordXRepositories** [comment]
I don't really agree removing the content, yes sticky is problematic in ways like drama, but you can still learn from the tutorial that he made. You can like the content without liking the author- for example, the harry potter books. J.K. Rowling may have made some bad comments, but the books are still good, i like them.

**Comment by Gankutsuou-Gank** [comment]
> I don't really agree removing the content, yes sticky is problematic in ways like drama, but you can still learn from the tutorial that he made. You can like the content without liking the author- for example, the harry potter books. J.K. Rowling may have made some bad comments, but the books are still good, i like them.

There is alot of other tutorials why use the one made by a bad person

**Comment by ColdlordXRepositories** [comment]
> > I don't really agree removing the content, yes sticky is problematic in ways like drama, but you can still learn from the tutorial that he made. You can like the content without liking the author- for example, the harry potter books. J.K. Rowling may have made some bad comments, but the books are still good, i like them.
> 
> 
> 
> There is alot of other tutorials why use the one made by a bad person

The fnf devs didn't know, it was just popular, so they grabbed it.
Hell will not rise from this. Don't worry.

**Comment by Gankutsuou-Gank** [comment]
> > > I don't really agree removing the content, yes sticky is problematic in ways like drama, but you can still learn from the tutorial that he made. You can like the content without liking the author- for example, the harry potter books. J.K. Rowling may have made some bad comments, but the books are still good, i like them.
> > 
> > 
> > There is alot of other tutorials why use the one made by a bad person
> 
> The fnf devs didn't know, it was just popular, so they grabbed it. Hell will not rise from this. Don't worry.

What do you think is more logical 
- supporting someone who is a terrible person
- supporting a smaller creator that is unknown

**Comment by Gankutsuou-Gank** [comment]
Plus the sticky situation thing breaks the Code of Conduct standards

**Comment by ninjamuffin99** [comment]
> Plus the sticky situation thing breaks the Code of Conduct standards

What I was referring to was if the CONTENT itself that got linked breaks the Code of Conduct, although I will also say it would be generally up to discretion of THE PEOPLE a bit (which of course has it's own issues). But ideally this place should be generally self policing and whatnot. What makes people prefer this tutorial over other ones, etc. etc.

Also I don't think it's too worthwhile to make the argument of "support someone small and unknown" since I'd like to hope that is irrelevant. These resources shouldn't be an underrated collection of niche things, it should be things that are 100% USEFUL and PRACTICAL that are GOOD. Whether it's made by big crew, or single person with 3 follows! I don't want that to be anyone's intention wit stuff when adding links! Just things that are good and cool and useful!

I will say in it's current state, this list isn't an exact "we support all the people here!" list, it's more of "here are things that FNF community people find very useful or interesting". just little disclaimer about intentions 🙇‍♂️

**Comment by ninjamuffin99** [comment]
i poked thru both videos and I think they are BOTH helpful and useful, so in my opinion, I don't think one is replaceable for the other, and hopefully it's not too touchy of a decision, anyone genuinely hurt by this decision, write me an email to cameron.taylor.ninja@gmail.com. 

I cannot guarantee to get back to you but I like a bit of a gauge of where people are at. Even discourse can be fussy!

**Comment by BombasticTom** [comment]
> > Plus the sticky situation thing breaks the Code of Conduct standards
> 
> What I was referring to was if the CONTENT itself that got linked breaks the Code of Conduct, although I will also say it would be generally up to discretion of THE PEOPLE a bit (which of course has it's own issues). But ideally this place should be generally self policing and whatnot. What makes people prefer this tutorial over other ones, etc. etc.
> 
> Also I don't think it's too worthwhile to make the argument of "support someone small and unknown" since I'd like to hope that is irrelevant. These resources shouldn't be an underrated collection of niche things, it should be things that are 100% USEFUL and PRACTICAL that are GOOD. Whether it's made by big crew, or single person with 3 follows! I don't want that to be anyone's intention wit stuff when adding links! Just things that are good and cool and useful!
> 
> I will say in it's current state, this list isn't an exact "we support all the people here!" list, it's more of "here are things that FNF community people find very useful or interesting". just little disclaimer about intentions 🙇‍♂️

Speaking of useful resources, I know a guy on youtube who makes great flixel tutorials. I'll probably make a pr as soon as i get home from school tomorrow (if he doesn't get added in that time)

**Comment by Gankutsuou-Gank** [comment]
> i poked thru both videos and I think they are BOTH helpful and useful, so in my opinion, I don't think one is replaceable for the other, and hopefully it's not too touchy of a decision, anyone genuinely hurt by this decision, write me an email to [cameron.taylor.ninja@gmail.com](mailto:cameron.taylor.ninja@gmail.com).
> 
> I cannot guarantee to get back to you but I like a bit of a gauge of where people are at. Even discourse can be fussy!

It's not really a big deal lol 
But I just feel bad for Alex (the victim) because they have seen alot of shit because of that guy, they even tried to commit suicide because of him,
anyways a small tutorial won't hurt lol
